Common/OpenCL/ITKimprovements/itkOpenCLGlobal.h

```diff
@@ -26,8 +26,8 @@ template< typename T >
 static inline T *
 OpenCLGetPtrHelper( T * ptr ) { return ptr; }
 template< typename TObjectType >
-static inline typename std::auto_ptr< TObjectType >::element_type * OpenCLGetPtrHelper(
-  const std::auto_ptr< TObjectType > &p )
+static inline typename std::unique_ptr< TObjectType >::element_type * OpenCLGetPtrHelper(
```

Comment:
Isn't `typename std::unique_ptr< TObjectType >::element_type` exactly the same as `TObjectType`? Anyway, your change from `auto_ptr` to k`unique_ptr' looks fine to me.